src/Components/Reviews.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+
+import ReviewsFilter from "./ReviewsFilter";
+import { fetchAllReviews } from "../api";
+import ReviewsCard from "./ReviewsCard";
+
+export default function Reviews() {
+  const [reviewData, setReviewData] = useState([]);
+
+  useEffect(() => {
+    fetchAllReviews().then((reviews) => setReviewData(reviews));
+  }, []);
```

Comment:
add some loading state and functionality please

Reply:
done